StudCity.Frontend/src/UI/GoogleButton.jsx

```diff
@@ -5,42 +5,44 @@ import { useDispatch } from 'react-redux';
 import { useGoogleLogin } from '@react-oauth/google';
 import { googleLogin } from '../app/features/authenticate.api';
 import { showAlert } from '../services/showAlert';
+import { useTranslation } from 'react-i18next';
 
 function GoogleButton({ className = '' }) {
-    const dispatch = useDispatch();
-    const navigate = useNavigate();
+	const { t } = useTranslation();
+	const dispatch = useDispatch();
+	const navigate = useNavigate();
 
-    const login = useGoogleLogin({
-        // eslint-disable-next-line camelcase
-        onSuccess: ({ access_token }) => {
-            dispatch(googleLogin(access_token)).then(({ payload }) => {
+	const login = useGoogleLogin({
+		// eslint-disable-next-line camelcase
```

Comment:
do we need this comment?

Reply:
Google returns the object key in this format, and our linter does not accept this, so there must be this comment